docs/quick-setup.md

```diff
@@ -0,0 +1,132 @@
+
+## Prerequisites
+> **Warning**
+> For the quick start to work, you must clone all the service repositories into the same directory as shown below
+
+```bash
+── microservices_march
+    ├── messenger
+    ├── notifier
+    └── platform <-- you are here
+```
+
+For example:
+```bash
+mkdir microservices_march
+
+cd microservices_march
+
+git clone git@github.com:microservices-march/messenger.git
+git clone git@github.com:microservices-march/notifier.git
+git clone git@github.com:microservices-march/platform.git
+
+cd platform
+```
+
+All the following commands assume that you have this directory structure. The quick start will not work if you deviate.
+The name of the parent directory can be whatever you'd like.
+
+## Starting the Demo Architecture
+The following command will start all the services and shared infrastructure:
+
+```bash
+docker-compose -f docker-compose.full-demo.yml up --abort-on-container-exit -d
+```
+> **Note**
+> See that we set `--abort-on-container-exit`. This is because we need all services to be up and running, but docker-compose will happily start up just the services it can if you do not pass this flag. This can lead to confusing failures.
+
+At this point, all the services, their databases, and the shared infrastructure is up and running. However, we need to set up the database schemas and sample data for the services.
+
+## Set up the Databases
+Run the following commands to prepare the databases for the `messenger` and `notifier` applications.
+
+You should only need to do this once, but you can do it again if you want to "reset" the data back to a clean state since these commands completely recreate the database.
+
+
+## `messenger` DB Setup
+```bash
+# This script creates the databse in postgres
+docker-compose exec -e PGDATABASE=postgres messenger node bin/create-db.mjs
+
+# This script sets up the tables, constraints, and indexes
+docker-compose exec messenger node bin/create-schema.mjs
+
+# This script writes seed data to the tables.
+docker-compose exec messenger node bin/create-seed-data.mjs
+```
+
+## `notifier` DB Setup
+```bash
+# This script creates the databse in postgres
+docker-compose exec -e PGDATABASE=postgres notifier node bin/create-db.mjs
+
+# This script sets up the tables, constraints, and indexes
+docker-compose exec notifier node bin/create-schema.mjs
+
+# This script writes seed data to the tables.
+docker-compose exec notifier node bin/create-seed-data.mjs
+```
+
+## Verify it's Working
+Now start tailing the `notifier` logs to see notifications:
+```bash
+docker-compose logs -f notifier
+```
+
+Now you can create a conversation between two users and send some messages.  You should see information about the notifications going out in the notifier logs.
+
+```bash
+# Create a conversation between user 1 and user 2
+curl -d '{"participant_ids": [1, 2]}' -H "Content-Type: application/json" -X POST http://localhost:80/conversations
```

Comment:
Yeah I normally prefer no quotes but it's necessary if you have query params.  Let's go with quotes since if we ever need to add query params to these they will be consistent.